response_parsing.go

```diff
@@ -41,6 +50,17 @@ type MemberData struct {
 	UserInfo map[string]string `json:"user_info,omitempty"`
 }
 
+func unmarshalledTriggerChannelsList(response []byte) (*TriggerChannelsList, error) {
+	channels := &TriggerChannelsList{}
+	err := json.Unmarshal(response, &channels)
```

Comment:
Seems weird to pass `**TriggerChannelsList` here. A bit surprised that works.

Reply:
Ah this was an oversight. I guess it's `json.Unmarshal` magic ✨